src/components/MentorBanner.tsx

```diff
@@ -0,0 +1,104 @@
+import { Box, Button, Typography, useMediaQuery } from '@mui/material';
+import Link from 'next/link';
+import React from 'react';
+
+import { LandingPageResponse } from '@utils/types';
+
+import theme from '../theme';
+
+interface BannerProps {
+  title: string;
+  description: string;
+  images: LandingPageResponse['fullBannerSection']['images'];
+  link: LandingPageResponse['fullBannerSection']['link'];
+}
+
+export const MentorBanner: React.FC<BannerProps> = ({
+  title,
+  description,
+  images,
+  link,
+}) => {
+  const isMobile = useMediaQuery(theme.breakpoints.down(600));
+
+  const { alt: desktopAlt, path: desktopPath } = images[0];
+  const { alt: mobileAlt, path: mobilePath } = images[1];
+  const { title: linkTitle, uri: linkUri, label: linkLabel } = link;
+
+  const image = isMobile ? mobilePath : desktopPath;
+  const alt = isMobile ? mobileAlt : desktopAlt;
+
+  return (
+    <Box
+      aria-label={alt}
+      data-testid="mentor-banner"
+      sx={{
+        display: 'grid',
+        justifyItems: 'center',
+        alignContent: 'space-evenly',
+        padding: '8%',
+        height: isMobile ? '374px' : '617px',
```

Comment:
This should be hardcoded? Would be better to create a global variable and re-use in case we have to change to something else later.